content/applications/sales/crm/track_leads/resellers.rst

```diff
@@ -0,0 +1,123 @@
+=========
+Resellers
+=========
+
+Within Odoo's *CRM* app, leads can be forwarded to resellers (or partners). Leads can be manually
+assigned, or can be automatically assigned based on the resellers designated *level*, and location.
+
+Configuration
+=============
+
+To utilize the reseller features, the *Resellers* module first needs to be installed. Navigate to
+the :menuselection:`Apps` application, and remove the :guilabel:`Apps` filter from the
+:guilabel:`Search...` bar. Then search for `Resellers`.
+
+.. image:: resellers/resellers-module.png
+   :align: center
+   :alt: The resellers module in Odoo.
+
+After the module is installed, navigate to the :menuselection:`CRM app`. Under the
+:menuselection:`Configuration` menu is a new section, titled *Resellers*.
+
+Partner levels
+==============
+
+Partner *levels* are used to differentiate between various resellers. To view the partner levels,
+navigate to :menuselection:`CRM app --> Configuration --> Partner Levels`.
+
+Three levels are created by default in the *CRM* app:
+
+- :guilabel:`Gold`
+- :guilabel:`Silver`
+- :guilabel:`Bronze`
+
+New levels can be added, as needed, and the existing levels can be edited and renamed, if desired.
+
+.. image:: resellers/partners-webpage.png
+   :align: center
+   :alt: An example of the partners webpage, displaying available partners by level and location.
+
+Level weight
+------------
+
+Level weight is used to decide the probability a partner to be assigned a lead or opportunity. The
+level weight should be a numerical value, greater than zero. If the weight is zero, no leads are
+assigned.
```

Comment:
For skimm-ability, how about adding instructions on getting to the Partner Levels page?

```suggestion
Level weight is used to decide the probability a partner to be assigned a lead or opportunity. To define a level weight, go to :menuselection:`(insert navigation instructions)` and click the desired partner level. Then, in the :guilabel:`Level Weight` field, assign a numerical value, greater than zero to it. If the weight is zero, no leads are
assigned.
```